### DISCUSSION TITLE
YouTube playback choppy with mpv support enabled

### DISCUSSION THREAD

**Discussion by qupear** [smplayer-dev/smplayer#830] [General]
Either with mpv or mplayer "Multimedia engine" setting of SMPlayer when I change speed of playback of YouTube video to 200%-300% it becomes hardly to watch. Mpv engine in this case is choppy and mplayer does something messy with sound. At same time when change support video sites to "Internal Youtube support" it is suitable to watch with high speedup but this way you need to manually scroll sponsor portions of video. Is it some fundamental limitations of mpv and mplayer?

For example in Freetube app I was watching video at speed 200-300% easily and with SponsorBlock at same time. But Freetube now don't play video to me it just work as search and copy links. But in SMPlayer I found only one way of using SponsorBlock with lua scripts which is not working with "Internal Youtube support".

Is there any way to enable sponsorblock and 300% youtube speedup in SMPlayer at same time without audio and video distortions?

**Comment by smplayer-dev** [comment]
That's strange because "Internal Youtube support" uses mpv as well. Just in case try to turn off the "Use adaptive streams" option in preferences -> network.

**Reply by qupear** [reply]
"Use adaptive streams" option is disabled. When supporting video sites = mpv streams option is even greyed out. The video, even with 240p quality, plays for about 15 seconds, then stops for about a second like because of slow network or hardware and playback is always stuttering like this. This should not be slow network because I could watch youtube 2K no problem in browser and hardware is good enough. If "Internal Youtube support" uses mpv should SponsorBlock work? To me it doesn't skip sponsors only mpv does this.
